x-pack/docs/en/security/authentication/security-domain.asciidoc

```diff
@@ -0,0 +1,85 @@
+[role="xpack"]
+[[security-domain]]
+=== Security domains
+
+beta::[]
+
+Security domains are a method of grouping <<realms,realms>> to enhance resource 
+sharing and check ownership for the same user across different realms.
+
+Users can create various resources in {es} that they own, access, and manage.
+For example, using the <<async-search,async search API>> to create a search 
+request, <<security-api-create-api-key,creating an API key>>, or activating a 
+<<user-profile,user profile>>.
+
+When a user attempts to access a resource, {es} compares a user's `username`
+and the realm they're authenticating from to determine who they are. In some
+cases, the same user can authenticate with different realms. For example,
+an <<ldap-realm,LDAP realm>> and a <<saml-realm,SAML realm>> can be backed by
+the same directory service. Similarly, a SAML realm might
+<<configuring-authorization-delegation,delegate authorization>> to a
+<<native-realm,native realm>>.
```

Comment:
@ywangd Is this supposed to refer to the fact that users authn to a SAML realm which delegates authorization to an LDAP realm, are viewed as different users if they would directly authn to the LDAP realm, because I don't believe this is the case.